b2c_authentication.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+/**
+ * Plugin Name: b2c_authentication
+ * Plugin URI: https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect
+ * Description: A plugin that allows users to log in using B2C policies
+ * Version: 1.0
+ * Author: Olena Huang
+ */
+
+////////////////////////////////////////////////////////////////////////////////
+
+// Turn on error reporting, for debugging
+error_reporting(E_ALL);
```

Comment:
This should be removed. Debugging can be achieved by adding `define( 'WP_DEBUG', true );` to the dev blog's `wp-config.php`, as described in [https://codex.wordpress.org/Debugging_in_WordPress](https://codex.wordpress.org/Debugging_in_WordPress).

I personally like to use `define( 'WP_DEBUG_LOG', true );`, in combination with [Debug Bar](https://wordpress.org/plugins/debug-bar/) to see logs written to file.